src/ts/components/seekbar.ts

```diff
@@ -249,7 +251,9 @@ export class SeekBar extends Component<SeekBarConfig> {
         // paused nor playing. Playback updates are handled in the Timeout below.
         if (this.config.smoothPlaybackPositionUpdateIntervalMs === SeekBar.SMOOTH_PLAYBACK_POSITION_UPDATE_DISABLED
           || forceUpdate || player.isPaused() || (player.isPaused() === player.isPlaying())) {
-          this.setPlaybackPosition(playbackPositionPercentage);
+          if (!this.isSeeking()) {
```

Comment:
this can be combined with the if above

Reply:
I wanted to keep it separate, as there are already enough If statements above.

I will move the above to a separate variable and not have a nested if.